tests/artifacts/test_artifacts.py

```diff
@@ -466,3 +466,19 @@ def test_inline_body():
     )
     assert artifact.spec.get_body() == "123"
     assert artifact.metadata.key == "y"
+
+
+def test_import_artifact_using_relative_path():
```

Comment:
was the test failing *before* your changes?

Reply:
![image](https://user-images.githubusercontent.com/62285310/211747646-d8411d48-ef1f-4ba0-850e-7d46d5cbc482.png)
sure